python/smashbox/utilities/__init__.py

```diff
@@ -282,11 +282,17 @@ def webdav_propfind_ls(path):
 def webdav_delete(path):
     runcmd('curl -k %s -X DELETE %s '%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path)))
 
-def webdav_mkcol(path,silent=False):
+def webdav_mkcol(path,silent=False,user_num=None):
     out=""
     if silent: # a workaround for super-verbose errors in case directory on the server already exists
         out = "> /dev/null 2>&1"
-    runcmd('curl -k %s -X MKCOL %s %s'%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path),out))
+    runcmd('curl -k %s -X MKCOL %s %s'%(config.get('curl_opts',''),oc_webdav_url(remote_folder=path,user_num=user_num),out))
+
+def webdav_upload (file_to_upload, remote_dir, remote_filename, user_number):
+
+    url = "%s/%s" % (oc_webdav_url(remote_folder=remote_dir, user_num=user_number),remote_filename)
+    runcmd('curl -k %s -T %s %s ' % (config.get('curl_opts',''),file_to_upload, url))
```

Comment:
I'm fine leaving this out of these tests, but this should be fixed the sooner the better